lib/pliny/error_reporters/rollbar.rb

```diff
@@ -18,9 +18,11 @@ def notify(exception, context:, rack_env:)
       private
 
       def fetch_scope(context:, rack_env:)
-        {
-          request: proc { extract_request_data_from_rack(rack_env) }
-        }
+        scope = {}
+        if rack_env.respond_to?(:body)
```

Comment:
It's more that this is the method that is called internally so I figure a duck type check is sufficient